Lib/test/test_deque.py

```diff
@@ -269,6 +269,13 @@ def test_getitem(self):
         self.assertRaises(IndexError, d.__getitem__, 0)
         self.assertRaises(IndexError, d.__getitem__, -1)
 
+    def test_bool(self):
+        d = deque('abc')
+        self.assertEqual(d, True)
+
+        d = deque([])
+        self.assertEqual(d, False)
+
```

Comment:
we don't add tests inside `Lib/test`. To add our original tests, please do it into `extra_tests/snippets`